### DISCUSSION TITLE
Admin Access

### DISCUSSION THREAD

**Discussion by praveenaik008** [community/community#42456] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi 

I know how to add a user or member in repository level, But I own an organization and I want to add a user in admin level so that he or she can manage the entire organization level things.

How can I do that?

**Comment by jfullstackdev** [comment, marked as answer]
at the People tab 

![image](https://user-images.githubusercontent.com/47092464/209335953-879a9e96-a3dc-42fe-a1ee-2fd7988f9163.png)

then change role

**Comment by pishenica7810** [comment]
im sorry to desturb your attention ,work i see massege from support on git hub and stay afraid and delete evrything....
but i`d like to writing calm down i wasn`t understand whare is my mistake ...
thank you on your help and support

**Reply by pishenica7810** [reply]
please write me on gurlzy89@gmail.com thank for your attention and kindness advice

**Comment by Divya30-Jain** [comment]
Hi GitHub Support,

I recently noticed that I have admin access to the following repository/organization:


Could you please help me identify:
When this admin access was granted.
Who granted me the admin permission.
Not able to find anything from audit logs
